### PR TITLE
Feature/constants from config

### PR DESCRIPTION
**Pull request by siddharthanpr**
This Uses the config file to directly load hyperparameters from it instead of manual copies 
DET-2346

### CODE REVIEW

**Review by ramchandra-karandikar — APPROVED** [review]
Looks good

**Comment by ramchandra-karandikar** [comment]
It would be great to include 'max_num_points' as another field inside Configuration file, but that can be subject
of another PR

**Comment by siddharthanpr** [comment]
> It would be great to include 'max_num_points' as another field inside Configuration file, but that can be subject of another PR

it is noe exported as a part of the input binding shape. When it becomes dynamic, we will have to figure things out. And also at that point we wont have the num voxel parameter I think

File: mmdet3d/apis/trt_engine.py [hunk 1]
```diff
@@ -3,22 +3,31 @@
 import tensorrt as trt
 import pycuda.autoinit
 import pycuda.driver as cuda
+from configs.centerpoint import centerpoint_03pillar_kitti_lum as centerpoint_config
```
Comment (siddharthanpr) [inline mmdet3d/apis/trt_engine.py:6]: @meremeev Can you tell me if this is ok? This means wee will have to run trt_engine.py within opnmmlab conda environment.

**Pull request merged**